src/apps/answers/api.py

```diff
@@ -839,8 +839,8 @@ async def applet_validate_multiinformant_assessment(
     code = None
     try:
         await AppletService(session, user.id).exist_by_id(applet_id)
-        await CheckAccessService(session, user.id).check_applet_manager_list_access(applet_id)
-
+        await CheckAccessService(session, user.id).check_applet_manager_and_limited_list_access(applet_id)
+  
         await AnswerService(session, user.id, answer_session).validate_multiinformant_assessment(
```

Comment:
We don't need to check if user is OWER or MANAGER, just check is a RESPONDENT in applet if enough

```
await CheckAccessService(session, user.id).check_answer_check_access(applet_id)
```

Reply:
Use check_answer_check_access() method that already exists